lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -380,24 +380,18 @@
                 createTeamsListView: function(options) {
                     var topic = options.topic,
                         collection = options.collection,
-                        myTopicTeamsCollection = new TeamCollection(
-                            this.context.userInfo.teams,
-                            {
-                                teamEvents: this.teamEvents,
-                                course_id: this.context.courseID,
-                                username: this.context.userInfo.username,
-                                topic_id: topic.get('id'),
-                                perPage: 5,
-                                parse: true,
-                                url: this.context.myTeamsUrl
+                        topicTeams = _.filter(
```

Comment:
If you do end up going this way, I don't know that there's much value in keeping it as a list, there should really only ever be one or zero.